Week01/Problem05/utantipongpipat_05.py

```diff
@@ -0,0 +1,11 @@
+import math;
+
+def lcm(a,b):
+    return a//math.gcd(a,b)*b;
```

Comment:
Nice use of existing Python functionality.  In many cases, using someone else's code (by importing a module which has the functionality I wanted) has saved me hours, if not days, of work trying to write the code myself.